src/main/scala/wdl4s/formatter/SyntaxFormatter.scala

```diff
@@ -79,8 +79,8 @@ class SyntaxFormatter(highlighter: SyntaxHighlighter = NullSyntaxHighlighter) {
   }
 
   private def formatImport(imp: Import): String = {
-    val namespace = imp.namespaceTerminal.map(ns => s" as ${ns.sourceString}").getOrElse("")
-    s"${highlighter.keyword("import")} '${imp.uri}'$namespace"
+    val namespace = imp.namespaceName
+    s"${highlighter.keyword("import")} '${imp.uri}' as $namespace"
```

Comment:
ToL: Hmmm, this is used when highlighting source code? I think it's potentially a little confusing if we're return edited source code instead?